catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -208,4 +230,38 @@ public void setFeatureService(FeatureService featureService) {
   public void setEndpointUtil(EndpointUtil util) {
     this.util = util;
   }
+
+  private Map<String, Object> getLatLonSuggestions(String query) {
+    Matcher matcher = latLonPattern.matcher(query);
+    List<Double> numbers = new ArrayList<>();
+    while (matcher.find()) {
+      numbers.add(Double.parseDouble(matcher.group()));
+    }
+    if (numbers.isEmpty() || numbers.size() == 1) {
+      return null;
+    }
+    if (numbers.size() % 2 != 0) {
+      numbers.remove(numbers.size() - 1);
+    }
```

Comment:
❓ Should we log that the query format was imperfect in the two conditionals above?